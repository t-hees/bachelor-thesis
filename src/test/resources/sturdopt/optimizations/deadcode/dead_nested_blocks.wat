(module
  (func (export "main") (result i32)
    (block (result i32)
      (block (result i32)
        i32.const 1
        (if (result i32)
          (then
            i32.const 1
            br 2
            i32.const 0
          )
          (else
            i32.const 1
          )
        )
        unreachable
      )
    )
  )
)
