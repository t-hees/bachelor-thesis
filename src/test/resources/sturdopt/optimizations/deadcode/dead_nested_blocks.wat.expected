(module
  (func (export "main") (result i32)
    (block (result i32)
      (block (result i32)
        i32.const 1
        drop
        (block (result i32)
          i32.const 1
          br 2
        )
      )
    )
  )
)
